apps/jobs/views.py

```diff
@@ -1361,6 +1361,79 @@ def delete_submission_message_from_queue(request, queue_name):
         return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
 
 
+@swagger_auto_schema(
+    methods=["get"],
+    manual_parameters=[
+        openapi.Parameter(
+            name="queue_name",
+            in_=openapi.IN_PATH,
+            type=openapi.TYPE_STRING,
+            description="Queue Name",
```

Comment:
Get count of the pending message in a queue.

Reply:
This is the description for the parameters list. Do you mean that I should change the description to the API function ?

Reply:
Oh..my bad. Thanks for correcting me.